roles/fips/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: 'Install dracut-fips'
+  action: yum name=dracut-fips state=installed
+
+- name: 'Run dracut to rebuild initramfs'
+  command: dracut --force
+
+- name: 'Edit kernel command-line to include the fips=1 argument'
+  command: grubby --update-kernel=DEFAULT --args=fips=1
+
+- name: 'Get uuid'
+  command: 'findmnt -no uuid /boot'
+  register: result
+
+- name: 'Edit kernel command-line to include the boot=UUID=XXXX or boot=LABEL=XXXX or boot=/dev/DEVICE argument'
+  command: "grubby --update-kernel=DEFAULT --args=boot=UUID={{result.stdout}}"
+
+- name: Restart server
```

Comment:
Should we add a step to wait for the server to come back up before exiting? We could then add in the check to output whether FIPs was properly enabled or not rather than having it as a manual user step.

Reply:
have tried `wait_for` module over here but didn't find any success, its going timeout to reach the machine after reboot, any help over this @ehelms

Reply:
I think this is a good case for a `handler` in Ansible parlance, and then attaching that handler to the tasks above so that if they change the handler to reboot is initiated. If nothing changes, no reboot.

Reply:
Have you tried this pattern? https://www.linux.com/blog/ansible-reboot-server-play-book-and-wait-it-come-back